gnucash/import-export/import-backend.c

```diff
@@ -387,8 +387,25 @@ tokenize_string(GList* existing_tokens, const char *string)
     /* add each token to the token GList */
     while (stringpos && *stringpos)
     {
-        /* prepend the char* to the token GList */
-        existing_tokens = g_list_prepend(existing_tokens, g_strdup(*stringpos));
+        /* check for empty strings */
```

Comment:
Leave out the comments that state the obvious.

Reply:
Done